app/layout.tsx

```diff
@@ -1,13 +1,13 @@
 import './global.css'
 
 export default function RootLayout({
-  children,
+    children,
 }: {
-  children: React.ReactNode
+    children: React.ReactNode
 }) {
-  return (
-    <html lang="en">
-      <body>{children}</body>
-    </html>
-  )
-}
+    return (
+        <html lang="en">
+            <body>{children}</body>
+        </html>
+    )
+}
```

Comment:
It is a good practice to have an empty line at the end of the file. Without it we have this red sign